core/configuration/tmux.conf

```diff
@@ -0,0 +1,8 @@
+# Basic configuration
+set-option -g history-limit 10000
+
+# Enable mouse over splits
+set -g mouse on
+
+# Force utf-8 usage for non ASCII characters
+setw -gq utf8 on
```

Comment:
what does this do in practice?

Reply:
Turning utf8 on solves some problems when running tmux under windows terminal and others, so if someone is using it and tmux is the default endpoint, we should at least enable to avoid any related problem.